lib/dora-parser/src/lexer.rs

```diff
@@ -582,6 +588,7 @@ fn keywords_in_map() -> HashMap<&'static str, TokenKind> {
     keywords.insert("static", TokenKind::Static);
     keywords.insert("spawn", TokenKind::Spawn);
     keywords.insert("const", TokenKind::Const);
+    keywords.insert("@", TokenKind::At);
```

Comment:
Sorry, didn't see this yesterday: Could you please parse `@` as an operator instead (in `read_operator`).

Reply:
Will do.

Reply:
`read_operator` only returns the token itself, where should I parse the rest?

Reply:
Done.